odoo/addons/base/models/assetsbundle.py

```diff
@@ -183,9 +183,9 @@ def to_node(self, css=True, js=True, debug=False, async_load=False, defer_load=F
     @func.lazy_property
     def last_modified(self):
         """Returns last modified date of linked files"""
+        assets = [WebAsset(self, url=f['url'], filename=f['filename'], inline=f['content']) for f in self.files if f['atype'] in ['text/sass', "text/scss", "text/less", "text/css", "text/javascript"]]
```

Comment:
It may be easier to read in several lines.
```py
assets = [WebAsset(self, url=f['url'], filename=f['filename'], inline=f['content'])
    for f in self.files
    if f['atype'] in ['text/sass', "text/scss", "text/less", "text/css", "text/javascript"]]
```

Reply:
I agree, this is better. Thank's